lib/settings.ts

```diff
@@ -36,7 +37,8 @@ export const defaultSettings: ISettings = {
     preferredLogicTypePort: 'unresolved',
     preferredLogicTypeSignal: 'unresolved',
     unusedSignalRegex: '_unused$',
-    ieeeCasing: 'lowercase'
+    ieeeCasing: 'lowercase',
+    warnSpaceBeforeUnit: true
   },
```

Comment:
Maybe we just make a list with globally ignoreable rules? Than we could also make a ignore global rule thingy?

Reply:
Sounds good